fdbclient/StorageServerInterface.h

```diff
@@ -30,6 +30,7 @@
 #include "fdbrpc/Stats.h"
 #include "fdbrpc/TimedRequest.h"
 #include "fdbrpc/TSSComparison.h"
+#include "fdbclient/BlobGranuleCommon.h"
```

Comment:
Part of figuring out best location for MutationsAndVersionsRef. Ideally don't include blob stuff here